src/router/routes.ts

```diff
@@ -3,36 +3,59 @@
 // import StateExample from '@/views/StateExample.vue'
 import DefineCompany from '@/views/DefineCompany.vue'
 import ReviewConfirm from '@/views/ReviewConfirm.vue'
+import SigninView from '@/views/auth/Signin.vue'
+import SignoutView from '@/views/auth/Signout.vue'
 
 export const routes = [
   {
     path: '/',
     redirect: '/define-company'
   },
-  // {
-  //   path: '/home',
-  //   name: 'home',
-  //   component: Home,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
-  // {
-  //   path: '/MixinExample',
-  //   name: 'mixinExample',
-  //   component: MixinExample,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
-  // {
-  //   path: '/StateExample',
-  //   name: 'stateExample',
-  //   component: StateExample,
-  //   meta: {
-  //     requiresAuth: false
-  //   }
-  // },
+  {
+    path: '/signin/:idpHint',
+    name: 'signin',
+    component: SigninView,
+    props: true,
+    meta: {
+      requiresAuth: false
+    }
+  },
+  {
+    path: '/signin/:idpHint/:redirectUrl',
```

Comment:
Is it possible in the PR to remove the other 2 signin routes? (Per my RC text where I noted that only the second route is used -- the one with 2 route parameters.)